modules/user_accounts/ajax/getData.php

```diff
@@ -0,0 +1,141 @@
+<?php
```

Comment:
@driusan @alisterdev Okay, I like the sounds of that; I will look into it. Do we have this used for forms anywhere in Loris?